### PR TITLE
E-1050 feat(ui): add new fonts

### PR DESCRIPTION
**Pull request by ifiri**

### CODE REVIEW

**Comment by ifiri** [comment]
@dudiq ready for review

**Comment by dudiq** [comment]
@lamartire ready for approve

File: packages/ui/src/scss/typography/_fonts.brutal-type.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
+@font-face {
+  font-family: 'Brutal Type';
+  src: url('/fonts/BrutalType/brutal-type-regular.eot');
```
Comment (lamartire) [inline packages/ui/src/scss/typography/_fonts.brutal-type.scss:3]: Add `src: local('Brutal Type')` to all fonts for locally installed ones fallback

Comment (ifiri) [reply packages/ui/src/scss/typography/_fonts.brutal-type.scss:3]: commonly, we already can use only woff2, works everywhere except dead IE https://caniuse.com/#feat=woff2

Comment (ifiri) [reply packages/ui/src/scss/typography/_fonts.brutal-type.scss:3]: if dead IE support required, then woff2 + eot

@kazurus what's browser support we should maintain?

Comment (kazurus) [reply packages/ui/src/scss/typography/_fonts.brutal-type.scss:3]: @ifiri now it's ok, add only src: local('Brutal Type')

**Comment by ifiri** [comment]
@lamartire done

**Comment by lamartire** [comment]
@kazurus approved

**Pull request merged**